docker.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+RUST_TOOLCHAIN=1.46
+ARCH=$(uname -m)
+GIT_COMMIT=$(git rev-parse HEAD)
+GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+DOCKER_TAG=rustvmm/dev
+DOCKERHUB_LINK=https://hub.docker.com/r/${DOCKER_TAG}/tags
+
+# Get docker tags
+dtags () {
+    local image="${1}";
+    wget -c -q https://registry.hub.docker.com/v1/repositories/"${image}"/tags -O - | tr -d '[]" ' | tr '}' '\n' | awk -F: '{print $3}'
+}
+
+# Get the latest published version. Returns a number.
+# If latest is v100, returns 100.
+latest(){
+  dtags $DOCKER_TAG | grep -v "_" | cut -c 2- | sort -n | tail -1
+}
+
+# Builds the tag for the newest versions. It needs the last published version number.
+# Returns a valid docker tag.
+build_tag(){
+  latest_version=$(latest)
+  new_version=$(($latest_version + 1))
+  new_tag=${DOCKER_TAG}:${new_version}_$ARCH
+  echo $new_tag
+}
+
+# Build a new docker version.
+# It expects the latest version (a number) in input.
+# It will build a new docker image with tag latest version + 1
+# and will alias it with "latest" tag.
+build(){
+  new_tag=$(build_tag)
+  set -e
+  docker build -t $new_tag \
+        --build-arg RUST_TOOLCHAIN=${RUST_TOOLCHAIN} \
+        --build-arg GIT_BRANCH=${GIT_BRANCH} \
+        --build-arg GIT_COMMIT=${GIT_COMMIT} \
+        -f Dockerfile .
+  set +e
+  echo "Build completed for $new_tag"
+}
+
+# Creates and pushes a manifest
+manifest(){
+  new_tag=$(build_tag)
+  set -e
+  docker manifest create \
+        $new_tag \
+        "${new_tag}_x86_64" \
+        "${new_tag}_aarch64"
```

Comment:
`new_tag` already appends x86_64 and aarch64 to the name, so this results in: dev:12_x86_64_x86_64.

Reply:
Good catch! New behaviour:
```
docker manifest create federicoponzi/test:1 federicoponzi/test:1_x86_64 federicoponzi/test:1_aarch64
Manifest successfully created
docker manifest push federicoponzi/test:1
Manifest successfully pushed on DockerHub: https://hub.docker.com/r/federicoponzi/test/tags
```